stytch/shared/jwt.go

```diff
@@ -19,3 +24,19 @@ func ReservedClaim(key string) bool {
 	// Stytch-specific claims are scoped by a URL prefix.
 	return strings.HasPrefix(key, "https://stytch.com/")
 }
+
+func ValidateJWTToken(token string, staticClaims jwt.Claims, keyFunc jwt.Keyfunc, audience string, issuer string, fallbackIssuer string) error {
```

Comment:
[sand] I strongly dislike the number of params here. Would you mind creating a `ValidateJWTTokenParams struct` to make it more clear at the callsites?